docs/topics/modules/reviewing_publishing_changes.adoc

```diff
@@ -44,13 +44,18 @@ image::updated_app.png[Updated applications]
 
 //for hello world
 ifeval::["{context}" == "hello-world"]
-Congratulations! You have now created your first quickstart project in {osio}, used a work item to track your work, made changes to your project code, committed the changes to GitHub, and published the new version of your project.
+You have now completed the fourth task *Review and publish changes to your codebase* in the *Test Iteration*, ensure that you change the state of the work item to *Closed* using the *Plan* tab.
+
+Also close the feature *User should be able to easily launch a sample vertx application using a smooth developer work-flow* now that all its child tasks have been completed.
+
+Congratulations! You have now created your first quickstart project in {osio}, used the planner to track and execute your work, made changes to your project code, committed the changes to GitHub, and published the new version of your project.
```

Comment:
This gives me the creeps. I think we can now change 'Congratulations' to 'Well done' or something like that. WDYT?